.github/issue_template.md

```diff
@@ -0,0 +1,106 @@
+<!--
+Thanks for reporting issues back to ownCloud! This is the issue tracker of ownCloud, if you have any support question please check out https://owncloud.org/support
+
+This is the bug tracker for the Phoenix. Find other components at https://github.com/owncloud/core/blob/master/.github/CONTRIBUTING.md#guidelines
+
+For reporting potential security issues please see https://owncloud.org/security/
+
+To make it possible for us to help you please fill out below information carefully.
+
+Before reporting any issues please make sure that you're using the latest available version for your major branch (e.g. 9.0.x), see https://owncloud.org/changelog/
+--> 
+### Steps to reproduce
+1.
+2.
+3.
+
+### Expected behaviour
+Tell us what should happen
+
+### Actual behaviour
+Tell us what happens instead
+
+### Server configuration
+**Operating system**:
+
+**Web server:**
```

Comment:
"Core server version"

and a separate field for "Phoenix version" in case of separate deployment ?